src/components/constraints/horizontal/horizontal.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import vars from '../../../../materials/custom-properties';
 
-const getConstraintSyles = constraint => {
+export const getConstraintSyles = constraint => {
```

Comment:
I'm exporting this only for internal usage.
Perhaps it might make sense to extract this function into a separate module.